src/polymer-project.ts

```diff
@@ -244,6 +222,69 @@ class HtmlSplitter extends Transform {
       callback(null, file);
     }
   }
+
+  _splitScript(file: File, doc: parse5.ASTNode, filePath: string) {
+    const scriptTags = dom5.queryAll(doc, HtmlSplitter.isInlineScript);
+    for (let i = 0; i < scriptTags.length; i++) {
```

Comment:
`splitScript` needs to use a `for`-loop because it uses `continue` at line 236 for unrecognized script types, while `splitStyle` doesn't. I could either (1) refactor this function so that the loop types are the same, or (2) restore the original `for`-loop in `splitStyle` for consistency (i.e., undo cc93d2a57c8f8335246d6d5e179eb4911fcaee37).